### PR TITLE
Add Anchor connector

### PR DESCRIPTION
**Pull request by mre**
First draft of Anchor connector. Not tested yet as API endpoints need to be added first.

### CODE REVIEW

File: .github/workflows/docker-anchor.yml
Comment (woolfg) [inline .github/workflows/docker-anchor.yml:1]: should we deprecate this method of calling/building a docker image and integrate it using the DB cred approach. not sure if it makes sense to maintain two approaches to run it.

**Review by woolfg — COMMENTED** [review]
- I would suggest to deprecate the single fetcher approach and combine all fetchers using the connector_manager approach. this would have the advantage of merging the helper modules without copying it and maintaining just one approach and code base which can share all helper function.
- the integration in the connector_manager is missing

File: .github/workflows/docker-anchor.yml
Comment (mre) [reply .github/workflows/docker-anchor.yml:1]: I've added it for debugging, mostly. Let's add it and then remove all the Docker pipelines in one go in a bit. Makes more sense from a "changeset" perspective.

**Comment by mre** [comment]
* [x] Yes, let's deprecate the single fetcher approach and combine all fetchers using the connector_manager approach.
We can do that as a next step. For now, I've added the files for the single fetcher for completeness' sake.
* [x] Added the integration in the connector_manager.

Tested locally and imports all Anchor data for our test podcast. (No API Endpoints yet.)
This is ready for another round of review now.

**Pull request merged**